rotary/rules/rotary.smk

```diff
@@ -70,18 +71,17 @@ rule install_polypolish:
         touch {output.install_finished}
         """
 
-
 # TODO - does not check the HMM version, only ID. If the HMM version updates, it won't automatically re-download
 rule download_hmm:
     output:
-        hmm=os.path.join(config.get("db_dir"), "hmm", config.get("start_hmm_pfam_id") + ".hmm")
+        hmm=os.path.join(config.get("db_dir"), "hmm", START_HMM_NAME + ".hmm")
     log:
         "logs/download/hmm_download.log"
     benchmark:
         "benchmarks/download/hmm_download.txt"
     params:
         db_dir=os.path.join(config.get("db_dir"), "hmm"),
-        url="http://pfam-legacy.xfam.org/family/" + config.get("start_hmm_pfam_id") + "/hmm"
+        url=config.get("hmm_url")
     shell:
         """
         mkdir -p {params.db_dir}
```

Comment:
@jmtsuji So it's a double-edged sword. On the one hand, it would prevent specific attacks where an attacker could doppelganger as the server and send out malware as the hmm. On the other hand, it is more convenient if the server fails to get a new certificate. TI added it during debugging because they didn't bother to get a certificate for the legacy server. After all, it was temporary.

Reply:
My recommendation would be to leave it out unless we run into the issue again constantly in the future.

Reply:
See: https://chat.openai.com/share/930460eb-2687-4e57-b683-e06ec3074eee for more info.

Reply:
OK, let's leave it out unless we have issues with this again in the future. I'll make a commit to this branch with the change.